app/views/admin/loans/timeline/table/_timeline.html.slim

```diff
@@ -0,0 +1,11 @@
+.timeline-table
+  - if @loan.root_timeline_entry
+    - if @loan.root_timeline_entry.is_a?(ProjectGroup)
+      table.table
+        = render "admin/loans/timeline/table/group", entry: @loan.root_timeline_entry, is_first: true,
+          max_depth: @loan.root_timeline_entry.max_descendant_depth
+    / If user has created a step through the timeline list, the timeline table is incompatible
```

Comment:
Oh interesting. Can you make a note in the next issue in this series for us to fix that. Should be an easy fix, just need to remember to do it. This is fine for now. Good catch!